psalm-baseline.xml

```diff
@@ -1,381 +1,191 @@
-      <code>new static('The "readConcern" option cannot be specified within a transaction. Instead, specify it when starting the transaction.')</code>
-      <code>new static('The "writeConcern" option cannot be specified within a transaction. Instead, specify it when starting the transaction.')</code>
-      <code>new static('Write concern is not supported by the server executing this command')</code>
-    </UnsafeInstantiation>
+      <code><![CDATA[$options['session']]]></code>
+    </MixedAssignment>
   </file>
   <file src="src/GridFS/Bucket.php">
-    <DocblockTypeContradiction occurrences="4">
-      <code>! is_resource($destination)</code>
-      <code>! is_resource($destination)</code>
-      <code>! is_resource($source)</code>
-      <code>! is_resource($stream)</code>
-    </DocblockTypeContradiction>
-    <MissingReturnType occurrences="5">
-      <code>delete</code>
-      <code>downloadToStream</code>
-      <code>downloadToStreamByName</code>
-      <code>drop</code>
-      <code>rename</code>
-    </MissingReturnType>
-    <MixedArgument occurrences="2">
-      <code>$options['revision']</code>
-      <code>$options['revision']</code>
-    </MixedArgument>
-  </file>
-  <file src="src/GridFS/Exception/CorruptFileException.php">
-    <UnsafeInstantiation occurrences="4">
-      <code>new static(sprintf('Chunk not found for index "%d"', $expectedIndex))</code>
-      <code>new static(sprintf('Expected chunk to have index "%d" but found "%d"', $expectedIndex, $index))</code>
-      <code>new static(sprintf('Expected chunk to have size "%d" but found "%d"', $expectedSize, $size))</code>
-      <code>new static(sprintf('Invalid data found for index "%d"', $chunkIndex))</code>
-    </UnsafeInstantiation>
-  </file>
-  <file src="src/GridFS/Exception/FileNotFoundException.php">
-    <UnsafeInstantiation occurrences="2">
-      <code>new static(sprintf('File "%s" not found in "%s"', $json, $namespace))</code>
-      <code>new static(sprintf('File with name "%s" and revision "%d" not found in "%s"', $filename, $revision, $namespace))</code>
-    </UnsafeInstantiation>
-  </file>
-  <file src="src/GridFS/Exception/StreamException.php">
-    <UnsafeInstantiation occurrences="3">
-      <code>new static(sprintf('Downloading file from "%s" to "%s" failed. GridFS filename: "%s"', $sourceMetadata['uri'], $destinationMetadata['uri'], $filename))</code>
-      <code>new static(sprintf('Downloading file from "%s" to "%s" failed. GridFS identifier: "%s"', $sourceMetadata['uri'], $destinationMetadata['uri'], $idString))</code>
-      <code>new static(sprintf('Uploading file from "%s" to "%s" failed. GridFS filename: "%s"', $sourceMetadata['uri'], $destinationUri, $filename))</code>
-    </UnsafeInstantiation>
+    <MixedArgument>
+      <code><![CDATA[$options['revision']]]></code>
+      <code><![CDATA[$options['revision']]]></code>
+    </MixedArgument>
   </file>
   <file src="src/GridFS/ReadableStream.php">
-    <MixedArgument occurrences="2">
```

Comment:
That's nice they [removed the `occurrences` number](https://github.com/vimeo/psalm/pull/8967). That will reduce merge conflicts.